src/index.ts

```diff
@@ -226,11 +226,6 @@ export class StringTable extends Array {
   #encodings = new Map<string, Uint8Array>()
   #positions = new Map<string, number>()
 
-  constructor() {
```

Comment:
I am a bit worried than now StringTable constructor will construct an empty table without ''. This makes the library harder to use for the clients since the usual case is harder to write (StringTable.from(['']) and this is likely to cause mistakes (eg. `example.js` is wrong).